lib/fetch/dataURL.js

```diff
@@ -429,14 +434,8 @@ function forgivingBase64 (data) {
     return 'failure'
   }
 
-  const binary = atob(data)
-  const bytes = new Uint8Array(binary.length)
-
-  for (let byte = 0; byte < binary.length; byte++) {
-    bytes[byte] = binary.charCodeAt(byte)
-  }
-
-  return bytes
+  const buffer = Buffer.from(data, 'base64')
+  return new Uint8Array(buffer.buffer, buffer.byteOffset, buffer.byteLength)
```

Comment:
I don't see a problem with the change here. Because all we are doing is base64 decoding.
Changing this section has greatly improved performance.

Reply:
These are *very* unlikely to be equivalent.

Reply:
The only difference between buffer and atob is that atob only supports latin1.
So I have to do this hackly thing to get back to utf8.